index.js

```diff
@@ -15,11 +15,8 @@ var request = require('request'),
 
 module.exports = Quickbooks
 
-Quickbooks.REQUEST_TOKEN_URL          = 'https://oauth.intuit.com/oauth/v1/get_request_token'
-Quickbooks.ACCESS_TOKEN_URL           = 'https://oauth.intuit.com/oauth/v1/get_access_token'
```

Comment:
Existing clients may be relying on these fields for OAuth 1.

Reply:
These fields was never used in codes. Since all tokens are passed from constructor itself.

Reply:
Right, they're not referenced in the package itself, but they're effectively part of the public api, provided as a convenience for people to use them something [like this](https://github.com/mcohen01/node-quickbooks/blob/0f9bf0515b1e95c9c14f5da9e127bfda3744bba2/example/app.js#L41). If you remove them, you may be breaking existing code.

Reply:
I think thats in another `node-quickbooks` module for Quickbooks Accounting. How will it break in this module? Little confused

Reply:
`Quickbooks.ACCESS_TOKEN_URL` exists in the current version. If someone references `Quickbooks.ACCESS_TOKEN_URL` in their application code somewhere, and then they bump the version of `quickbooks` in package.json, then their code will break, right? Because the version they upgraded to doesn't have `Quickbooks.ACCESS_TOKEN_URL`. [Great explanation](https://www.youtube.com/watch?v=oyLBGkS5ICk) of the general idea.

Reply:
Yeah didn't think that perspective.